stream_test.go

```diff
@@ -0,0 +1,60 @@
+package openssl
+
+import (
+	"bytes"
+	"io"
+	"os"
+	"testing"
+)
+
+func NoError(t *testing.T, err error) {
```

Comment:
Please unexport. (And maybe call it `requireNoError` - or use `require.NoError` from testify)